tools/experiments/experiments.html

```diff
@@ -189,6 +193,15 @@
     <h1>AMP Experiments</h1>
   </section>
 
+  <section class="desc">
+    <h2>Version</h2>
+    <p>
+      This is version <span class="version">$internalRuntimeVersion$</span> of the AMP runtime.
+      For a list of commits contained in this release, click
+      <a target="_blank" href="https://github.com/ampproject/amphtml/releases/tag/$internalRuntimeVersion$">here</a>.
```

Comment:
This link will 404 if one opens this page from a localhost, since the RTV number is most likely not the same as an official release...

Reply:
I don't believe any of the other sections of the experiments page work when served from `localhost`. I think we're okay as long as https://cdn.ampproject.org/experiments.html links to a published release.